src/components/shared/card.tsx

```diff
@@ -20,17 +20,18 @@ import { SvgCamera, SvgVideo, SvgAudio } from '@guardian/src-icons';
 interface Props {
     relatedItem: RelatedItem;
     image: Option<Image>;
+
 }
 
-const listStyles = css`
+const listStyles = (format: Format): SerializedStyles => css`
     background: white;
     margin-right: ${remSpace[3]};
     flex: 0 0 15rem;
     height: 100%;
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-
+    border-top : 1px solid ${getPillarStyles(format.pillar).kicker};
```

Comment:
inverted for dark mode?

Reply:
Should i invert the changed `palette.labs[300]` as it has changed to?